demo/src/pages/interApplicationBusPublish/InterApplicationBusPublish.tsx

```diff
@@ -10,25 +10,51 @@ const codeExample = `import {useInterApplicationBusPublish, useInterApplicationB
 
 const TOPIC = "demo-topic";
 
+const formatNames = (names: string[]) => {
+    return ["["].concat(names.join(", ").concat("]"));
+};
+
 const Component = () => {
-    const [name, setName] = useState("John Smith");
-    const {data} = useInterApplicationBusSubscribe({ uuid: "*" }, TOPIC);
-    useInterApplicationBusPublish(TOPIC, name);
+    const [receivedNames, setReceivedNames] = useState([] as string[]);
```

Comment:
You can use a generic in useState rather than casing here (e.g. `useState<string[]>`)